README.md

```diff
@@ -81,7 +81,7 @@ function current_user = 'modules/user/lib/queries/user/current'
 ```
 
 ```
-function permissions = 'modules/user/lib/queries/user/get_permissions', user: user
+function permissions = 'modules/user/lib/queries/user/find_permissions', user: user
```

Comment:
how about lib/queries/users/permissions/find (to be consistent with https://github.com/Platform-OS/pos-module-user/pull/12/files#diff-3cbbf8df9f65dda7435d69c0a819ef1ab44e0755fe670fba23938ac887314ebbR8 ? )